src/tls/openssl/tls.c

```diff
@@ -1195,6 +1196,8 @@ int tls_srtp_keyinfo(const struct tls_conn *tc, enum srtp_suite *suite,
 	memcpy(cli_key + key_size, p, salt_size); p += salt_size;
 	memcpy(srv_key + key_size, p, salt_size);
 
+	OPENSSL_cleanse(keymat, sizeof(keymat));
```

Comment:
Should we add our own simple secure clean variant in `mem/secure.c` like `mem_cleanse`? So a application can use this API too.

Reply:
In our case, we'd just use `OPENSSL_cleanse` directly or our own implementation.

Efficient implementation that involves `memset` can be messy, and I wouldn't want a naive one that does a bytewise loop with a `volatile` pointer.

Reply:
I've added a `mem_secclean` function.